apis/gear/src/TxBuilder.ts

```diff
@@ -0,0 +1,3 @@
+export class TxBuilder {
```

Comment:
Is it a placeholder for a functionality that will be available later?

Reply:
forgot to remove